core/dbt/adapters/factory.py

```diff
@@ -19,7 +19,7 @@
 
 
 class AdapterContainer:
-    def __init__(self):
```

Comment:
`mypy` is not supposed to throw an exception for `__init__()`. Is this currently happening?

Reply:
It doesn't throw an exception, it just complains about the fact that without a return value for init, it doesn't type check the type hints in Adaptercontainer, such as `self.adapters: Dict[str, Adapter] = {}`.

Reply:
awesome, ok